analyses/fusion-summary/01-fusion-summary.Rmd

```diff
@@ -151,36 +151,28 @@ specimensUnion<- union(arribaDF$tumor_id, starfusionDF$tumor_id)
 #### Write non-MB, non-ATRT embryonal fusions to file
 
 ```{r}
-allFuseEmbry <- allFuseEmbry %>%
-  prepareOutput(specimensUnion)
+if (!running_in_ci) {
+  allFuseEmbry <- allFuseEmbry %>%
```

Comment:
I don't really know what this means, but if it works 👍

Reply:
Before, we skipped the step for generating the fusions of interest file for EPN samples because they were not represented in the subset. Now I'm switching and running the EPN, skipping the non-ATRT, non-MB embryonal fusions and genes of interest because those are not represented but the EPN samples are.